samples/SocketsSample/wwwroot/hubs.html

```diff
@@ -78,6 +78,8 @@ <h1 id="head1"></h1>
 }
 
 let transportType = signalR.TransportType[getParameterByName('transport')] || signalR.TransportType.WebSockets;
+let hubRoute = getParameterByName('hubType') || "default";
+console.log("Hub Route: " + hubRoute);
```

Comment:
Wonder if we should integrate tslint into our build for things like this? https://palantir.github.io/tslint/rules/quotemark/

I can file an issue if there are major objections to the concept.